.vscode/tasks.json

```diff
@@ -7,12 +7,9 @@
 			"label": "Run Tests with Coverage",
 			"type": "shell",
 			"command": "go test -coverprofile=\"coverage.out\" && go tool cover -html=\"coverage.out\"",
-			"windows": {
-				"command": "go test -coverprofile=\"coverage.out\" & go tool cover -html=\"coverage.out\""
-			},
```

Comment:
any reason for removing this?

Reply:
Every time I ran that task it would not recreate the coverage output correctly, and it fails if there is no coverage file at all. So I end up using the linux version anyways - most people should install support for it if it's not already compatible. I also don't entirely know if it works sometimes, but that might just be me.